axelrod/fingerprint.py

```diff
@@ -293,9 +295,26 @@ def plot(self, col_map='seismic', interpolation='none', title=None):
         size = int((1 / self.step) // 1) + 1
         ordered_data = [self.data[point] for point in self.points]
         plotting_data = np.reshape(ordered_data, (size, size))
-        figure = plt.figure()
-        plt.imshow(plotting_data, cmap=col_map, interpolation=interpolation)
-        plt.axis('off')
+        fig, ax = plt.subplots()
+        cax = ax.imshow(plotting_data, cmap=col_map, interpolation=interpolation)
+
+        if colorbar:
+            max_score = max(ordered_data)
+            min_score = min(ordered_data)
+            ticks = [min_score, (max_score + min_score) / 2, max_score]
+            fig.colorbar(cax, ticks=ticks)
+
+        plt.xlabel('y')
+        plt.ylabel('x', rotation=0)
+        ax.xaxis.tick_top()
+        ax.xaxis.set_label_position('top')
+        ax.tick_params(axis=u'both', which=u'both', length=0)
+        plt.xticks([0, len(plotting_data) - 1], ['0', '1'])
+        plt.yticks([0, len(plotting_data) - 1], ['0', '1'])
+
+        if not labels:
```

Comment:
Can you add a test with `labels=False` and `colorbar=False` (just to check the plot runs).